docs/config.md

```diff
@@ -151,17 +151,17 @@ declare module 'abitype' {
 }
 ```
 
-### `StrictAbiType`
+### `Strict`
 
-When set, validates `AbiParameter`'s `type` against `AbiType`.
+When set, validates `AbiParameter`'s `type` against `AbiType` and enforces stricter checks on human readable ABIs.
 
 - Type `boolean`
 - Default `false`
 
 ```ts twoslash
 declare module 'abitype' {
   export interface Config {
-    StrictAbiType: false
+    Strict: false
```

Comment:
With this change it will affect type checking not only on types but also valid modifiers and parameter names. Kept as default so folks can hop in instead of out.